### DISCUSSION TITLE
Failed to run `config` for lualine.nvim; module 'trouble' not found

### DISCUSSION THREAD

**Discussion by santosh** [LazyVim/LazyVim#3836] [Q&A] [answered]
Here is the full error:

```
Failed to run `config` for lualine.nvim

...local/share/nvim/lazy/LazyVim/lua/lazyvim/plugins/ui.lua:202: module 'trouble' not found:
    no field package.preload['trouble']
cache_loader: module trouble not found
cache_loader_lib: module trouble not found
    no file './trouble.lua'
    no file '/build/nvim/parts/nvim/build/.deps/usr/share/luajit-2.1/trouble.lua'
    no file '/usr/local/share/lua/5.1/trouble.lua'
    no file '/usr/local/share/lua/5.1/trouble/init.lua'
    no file '/build/nvim/parts/nvim/build/.deps/usr/share/lua/5.1/trouble.lua'
    no file '/build/nvim/parts/nvim/build/.deps/usr/share/lua/5.1/trouble/init.lua'
    no file './trouble.so'
    no file '/usr/local/lib/lua/5.1/trouble.so'
    no file '/build/nvim/parts/nvim/build/.deps/usr/lib/lua/5.1/trouble.so'
    no file '/usr/local/lib/lua/5.1/loadall.so'

# stacktrace:
  - /LazyVim/lua/lazyvim/plugins/ui.lua:202 _in_ **values**
```

I'm on a fresh install of minimal Debian 12. I have installed `build-essential` already. Seems like some package is missing, and I'm not able to figure out what exactly. The same config runs on my other distros such as Ubuntu LTS, Arch.

Please guide me.

**Comment by dpetka2001** [comment, marked as answer]
You probably have `trouble.nvim` disabled?

**Reply by santosh** [reply]
For now, I'm going to mark this as answer.

I'm not sure why I disabled trouble.nvim at the first place.

**Reply by dpetka2001** [reply]
It's because you removed `if true then return {} end` from the `example.lua` file. That file is only meant to be read from the user as an example. Just create other files in your `/lua/plugins` directory if you want to put your customizations. That `example.lua` is quite old and has some configuration from older LazyVim versions, that I don't know if they'll work.
